packages/okidoc-md/src/buildMarkdown/buildMarkdown.spec.js

```diff
@@ -222,16 +222,28 @@ describe('buildMarkdown', () => {
       expect(markdown).toMatchSnapshot();
     });
 
+    // pending resolution: https://github.com/documentationjs/documentation/issues/1267
+    it.skip('should render markdown for interface with optional property', async () => {
+      const documentationSource = `
+     /** MyResult interface */
+      interface MyResult {
+        /** */
+        a?: string;
+      }
+    `;
+      const markdown = await getMarkdown(documentationSource, { title });
+
+      expect(markdown).toMatchSnapshot();
+    });
+
     it('should render markdown for methods interface as return type', async () => {
       const documentationSource = `
      /**
       * MyResult interface
-      * @property a - \`a\` description
-      * @property b - \`b\` description
```

Comment:
could you explain this change?

Reply:
Moved descriptions to properties declaration, so they will be documented.

Reply:
So it was documented with prev version of documentationjs and not documented with new one?:/

Reply:
Yes. They emptied `properties` list.